### DISCUSSION TITLE
How to return common data?

### DISCUSSION THREAD

**Discussion by Dindaleon** [async-graphql/async-graphql#1206] [Q&A]
I would like to return an access token with whatever request the client executes without specifying the access token field. How can I achieve this? 

Something like this: [Extensions](https://www.apollographql.com/docs/resources/graphql-glossary/#extensions)

**Comment by sunli829** [comment]
You can implement an extension youself, like this

https://github.com/async-graphql/async-graphql/blob/ee8b57bb3cbef2cd778418d0bd5d18572ba38608/src/extensions/analyzer.rs#L33
